app/services/sufia/workflow/abstract_notification.rb

```diff
@@ -32,7 +32,7 @@ def message
       private
 
         def users_to_notify
-          recipients[:to] + recipients[:cc]
+          recipients['to'] + recipients['cc']
```

Comment:
My experience was that `recipients['cc']` being empty causes this to fail with...an exception I can't recall of the top of my head.